yarn.lock

```diff
@@ -1429,10 +1429,9 @@ onetime@^5.1.0:
   dependencies:
     mimic-fn "^2.1.0"
 
-"opg-sirius-header@ministryofjustice/opg-sirius-header#semver:v0.26.0":
+"opg-sirius-header@ministryofjustice/opg-sirius-header#semver:v0.27.0-pfs-211-feature-flag-added-for-payments-admin.4":
```

Comment:
I think you'll need to run `yarn install` again as this is still pointing at the PR image.